hedera-node/src/main/java/com/hedera/services/legacy/services/state/AwareProcessLogic.java

```diff
@@ -111,10 +115,20 @@ public static boolean inSameUtcDay(Instant now, Instant then) {
 	public void incorporateConsensusTxn(Transaction platformTxn, Instant consensusTime, long submittingMember) {
 		try {
 			PlatformTxnAccessor accessor = new PlatformTxnAccessor(platformTxn);
-			if (!txnSanityChecks(accessor, consensusTime, submittingMember)) {
+			Instant timestamp = consensusTime;
+			if (accessor.getTxn().hasScheduleCreation() || accessor.getTxn().hasScheduleSign()) {
```

Comment:
Lets add new method on the `accessor` f.e `couldTriggerTransaction` and use it here:
`If (accessor.couldTriggerTransaction()) { .. }`. I think that it would be better not to hardcode the `op` types on this level.